Makefile

```diff
@@ -83,25 +75,6 @@ pypi: source-pypi develop
 install:
 	$(PYTHON) setup.py install --root $(DESTDIR) $(COMPILE)
 
-deb-prepare-source:
-	# build the source package in the parent directory
-	# then rename it to project_version.orig.tar.gz
-	dch -D "vivid" -M -v "$(VERSION)" "Automated (make builddeb) build."
-	$(PYTHON) setup.py sdist $(COMPILE) --dist-dir=../
-	rename -f 's/$(PROJECT)-(.*)\.tar\.gz/$(PROJECT)_$$1\.orig\.tar\.gz/' ../*
-
-deb-src: deb-prepare-source
-	# build the source package
-	dpkg-buildpackage -S -elookkas@gmail.com -rfakeroot
-
-deb-bin: deb-prepare-source
-	# build binary package
-	dpkg-buildpackage -b -rfakeroot
-
-deb: deb-prepare-source
-	# build both source and binary packages
-	dpkg-buildpackage -i -I -rfakeroot
-
```

Comment:
Actually I forgot to add the `contrib/packaging/debian/Makefile`.  Sending that in v2.